README.md

```diff
@@ -27,7 +27,14 @@ in [Wolt](https://wolt.com/) products, this UI component offers a visually
 appealing and highly customizable modal sheets with multiple pages, motion 
 for page transitions, and scrollable content within each page.
 
-- [Examples](#examples)
+- [Usage](#usage)
+  * [Basic Starter App](#basic-starter-app)
```

Comment:
Personally, I'm a big fan of this super basic starter example. Thank you so much for this!

Whenever I test a `wolt_modal_sheet` PR I don't have basic sample so i created my own [here](https://github.com/TahaTesser/wolt_modal_sheet_samples/blob/main/lib/basic_sample.dart) and now your PR solve this everyone which is absolutely wonderful.